qiskit/quantum_info/states/stabilizerstate.py

```
@@ -0,0 +1,591 @@
+            on all subsystems. If only some subsystems are reset this
+            function will perform a measurement on those subsystems and
+            evolve the subsystems so that the collapsed post-measurement
+            states are rotated to the 0-state. The RNG seed for this
+            sampling can be set using the :meth:`seed` method.
+        """
+        # Resetting all qubits does not require sampling or RNG
+        if qargs is None:
+            return StabilizerState(Clifford((np.eye(2 * self.data.num_qubits))))
+
+        for qubit in qargs:
+            # Apply measurement and get classical outcome
+            outcome = self._measure_and_update(qubit, 0)
+
+            # Use the outcome to apply X gate to any qubits left in the
+            # |1> state after measure, then discard outcome.
+            if outcome == 1:
+                _append_x(self.data, qubit)
+
+        return self
+
+    def measure(self, qargs=None):
+        """Measure subsystems and return outcome and post-measure state.
+
+        Note that this function uses the QuantumStates internal random
+        number generator for sampling the measurement outcome. The RNG
+        seed can be set using the :meth:`seed` method.
+
+        Args:
+            qargs (list or None): subsystems to sample measurements for,
+                                  if None sample measurement of all
+                                  subsystems (Default: None).
+
+        Returns:
+            tuple: the pair ``(outcome, state)`` where ``outcome`` is the
+                   measurement outcome string label, and ``state`` is the
+                   collapsed post-measurement stabilizer state for the
+                   corresponding outcome.
+        """
+        if qargs is None:
+            qargs = range(self.data.num_qubits)
+
+        outcome = ''
+        for qubit in qargs:
+            randbit = self._rng.randint(2)
```

I think randint is old style numpy rng and should raise an error with a `Generator`. I believe it should be `self._rng.integers(2)`

---

when I tried to change it, I got an error that `integers` is not a method of `numpy.random`, but `randint` works well.
maybe this should be updated in the `QuantumState` class?